### PR TITLE
Fix tests for 'update{}' section

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: src/tests/keywords/base64 [hunk 1]
```diff
@@ -71,12 +63,10 @@ if (&Tmp-String-9[0] != 'Og==') {
 	test_fail
 }
 
-&request <= {
```
Comment (alandekok) [inline src/tests/keywords/base64:74]: That's the new syntax tho... no `update` section.  It should be left alone.

File: src/tests/keywords/call [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #
-#  PRE: if
+#  PRE: update
```
Comment (alandekok) [inline src/tests/keywords/call:2]: If we're removing `update`, then none of the tests should depend on `update`

File: src/tests/keywords/call-empty [hunk 3]
```diff
@@ -14,6 +15,6 @@ if (&reply.Reply-Message[0] != "call second") {
 	test_fail
 }
 
-&reply -= &Reply-Message
+&reply.Reply-Message !* ANY
```
Comment (arr2036) [inline src/tests/keywords/call-empty:18]: Not equivalent?

File: src/tests/keywords/cast-integer [hunk 4]
```diff
@@ -8,9 +8,9 @@ if (<integer>&Class[0] == 257) {
 }
 
 if (<integer>&Class[0] < 256) {
-	test_fail
+	&reply.Filter-Id += "wrong"
 }
 
 if (<integer>&Class[0] > 257) {
-	test_fail
+	&reply.Filter-Id += "wrong"
```
Comment (arr2036) [inline src/tests/keywords/cast-integer:15]: Because why?

**Pull request closed without merging**